libs/books/data-access/src/lib/+state/reading-list.actions.ts

```diff
@@ -3,11 +3,11 @@ import { Book, ReadingListItem } from '@tmo/shared/models';
 
 export const init = createAction('[Reading List] Initialize');
 
-export const loadReadingListSuccess = createAction(
+export const loadReadingListSucceeded = createAction(
   '[Reading List API] Load list success',
   props<{ list: ReadingListItem[] }>()
 );
-export const loadReadingListError = createAction(
+export const loadReadingListFailed = createAction(
```

Comment:
Query: If you are updating action const, do you think we should change the action name as well to make it in sync?

This is applicable to all the action where you changed the const.

Reply:
Hi kandarp. 
3. Action naming should be consistent - action performing an event should use a past tense verb (like 'eventSucceeded').
I have consider it as a code smell and changed action names